src/constants.ts

```diff
@@ -37,6 +38,8 @@ export const THEME_TYPES = {
   DARK: 'dark',
 };
 
+export const AR = new Ar();
```

Comment:
0.0

Reply:
This is just a wrapper around big number - similar to our IOToken and mIOToken classes. (We should probably have a slightly abstracted version of those classes that are capable of being configured with different denominations, total supply, etc.